plugins/modules/azure_rm_rediscache.py

```diff
@@ -71,6 +71,42 @@
             - When set I(enable_non_ssl_port=true), the non-ssl Redis server port 6379 will be enabled.
         type: bool
         default: false
+    identity:
+        description:
+            - Identity for the WebApp.
+        type: dict
+        version_added: '2.7.0'
+        suboptions:
+            type:
+                description:
+                    - Type of the managed identity
+                choices:
+                    - SystemAssigned
+                    - UserAssigned
+                    - SystemAssigned, UserAssigned
+                    - None
+                default: None
+                type: str
+            user_assigned_identities:
+                description:
+                    - User Assigned Managed Identities and its options
+                required: false
+                type: dict
+                default: {}
+                suboptions:
+                    id:
+                        description:
+                            - List of the user assigned identities IDs associated to the WebApp
+                        required: false
+                        type: list
+                        elements: str
+                        default: []
+                    append:
+                        description:
+                            - If the list of identities has to be appended to current identities (true) or if it has to replace current identities (false)
+                        required: false
+                        type: bool
+                        default: False
```

Comment:
@p3ck  doc-default-does-not-match-spec: Argument 'append' in argument_spec found in identity -> user_assigned_identities defines default as (True) but documentation defines default as (False)